OneCommunityGlobal/HighestGoodNetworkApp
# Peterson fix the bug that causes the time log page to reload indefinitely

# Description
This pull request has been opened to fix two bugs: the first one causes the loading indicator to remain indefinitely on the time log page, and the second one removed the blue color and bold style from the hours in the Leaderboard table and the Total Time column.

## Related PRS (if any):
None
…

## Main changes explained:
The component Header has been modified to fix the bug that causes the loading indicator to remain indefinitely on the time log page.

The component Leaderboard has been modified to fix the bug that removed the blue color and bold style of the hours in the Leaderboard table and in RIGHT the Total Time column. If a person has INTANGIBLE TIME, that total will be different from the total on the left and it should also be bold and blue to make the difference obvious. 
…

## How to test:
1. Check into current branch
2. Do npm install and ... to run this PR locally
3. Clear site data/cache
4. Log any user
5 . Go to the time log and check if your ID was sent along with the URL.
6. The time log page should display a loading indicator, and after the page has successfully loaded, the loading indicator should disappear and a table should appear.
7.  Go to the dashboard page and check if the hours in the leaderboard table, in the total column, are displayed in blue color and bold style if the total intangible time is greater than zero.
![gif ](https://github.com/user-attachments/assets/1d58c047-f5df-489f-834b-38cbdaa84639)


## Screenshots or videos of changes:
Before my fix:
![Before my fix](https://github.com/user-attachments/assets/e13a760e-23a9-470a-81f2-607b1dd51e5f)

After my fix:
![After my fix](https://github.com/user-attachments/assets/6358c924-930a-4c36-b287-69fae000c73c)


## Note:
None.
